src/api/api.ts

```diff
@@ -0,0 +1,57 @@
+import { Product } from '../types/Product';
```

Comment:
I'd probably called this file `index.ts` and use it as a main entry point for all API calls. 

Then, create for each category: phones, tablets, all products just to have it nicely stored. 

Definitely, not a mistake here on your side as for current assignment, but as the app grows having all API calls in one file may become a bit of a pain.